fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_PerformancePage.js

```diff
@@ -806,7 +806,7 @@ class CDUPerformancePage {
         };
 
         let baroCell = "[\xa0\xa0\xa0]";
-        if (isFinite(mcdu.perfApprMDA)) {
+        if (mcdu.perfApprMDA !== null) {
```

Comment:
**question:** What is the reasoning behind this change?

Reply:
NaN is for the floating point unit, not for a placeholder value.